### DISCUSSION TITLE
Is there way to control the sequence between application

### DISCUSSION THREAD

**Discussion by gogovan-vincentngai** [argoproj/argo-cd#6153] [General]
May i know how to control the sequence between application..... ?
the sync-wave only control within the same application deploy sequence right ?

**Comment by wtam2018** [comment]
can you clarify what do you mean by sequence between applications?

**Reply by gogovan-vincentngai** [reply]
Yup, let say i have 5 argo application 

A - Create Namespace
B - Create Service account and rolebinding
C - Create Nginx Ingress
D - Create Cert-Manager 
E - Deploy Ingress Rules and Cert 


I would like them auto deploy in order A to E

**Comment by joebowbeer** [comment]
sync-wave is supposed to handle this. This article discusses an order similar to yours:

https://medium.com/lensesio/gitops-and-k8s-bootstrapping-752a1d8d7085

Docs: https://argoproj.github.io/argo-cd/user-guide/sync-waves/

There is an undiagnosed issue: https://github.com/argoproj/argo-cd/issues/5146
